README.md

```diff
@@ -34,15 +34,25 @@ Please refer to our [Contributing file](./.github/CONTRIBUTING.md)
 </p>
 
 
-### Members
 
-- [Bill Wagner](https://github.com/BillWagner)
-- [Khalid Abuhakmeh](https://github.com/khalidabuhakmeh)
-- [Jessica White](https://github.com/wordshaker)
+## Current Membership
+
+**Chair:**  [Jessica White](https://github.com/wordshaker)
+
+**Scribe** Madison Hardy
+
+**Members**
+- Eric Schneider
+- [Nicole Miller](https://github.com/nicolebuhakmeh)
 - [Ove Bastiansen](https://github.com/ovebastiansen)
-- [Mitchel Sellers](https://github.com/mitchelsellers)
+- [Aneesh Ambikavally](https://github.com/codehippie1)
+- Stacey Ramsey
 - [Calvin Allen](https://github.com/CalvinAllen)
+- [Joe Guadagno](https://github.com/jguadagno)
+- [Bill Wagner](https://github.com/BillWagner)
```

Comment:
I plan on focusing on the education committee. Happy to stay involved, if you want me to help out, but also fine to fade into the background.

Reply:
You aren't allowed to leave, I won't let you. 😄